addons/pos_sms/models/pos_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models
+
+
+class PosOrder(models.Model):
+    _inherit = 'pos.order'
+
+    def action_sent_message_on_sms(self, phoneNumber):
+        """ Send message on sms if sms is enabled and partner has mobile number or number is provided."""
```

Comment:
this comment does not seem accurate. It looks like `phoneNumber` has to be provided even if the partner has a phone number set.
The code is clear enough that you don't even need the comment so you can simply remove it.
This take on the subject is quite accurate: https://www.youtube.com/watch?v=Bf7vDBBOBUA